app/models/event.rb

```diff
@@ -0,0 +1,4 @@
+class Event < ApplicationRecord
+    enum player_types: [:single, :team]
```

Comment:
Is this actually neseccary or is Event going to have a `has_many: ;teams` association later anyways? Because if that is the case we could just implement `player_types = :single` by having a teamsize of 1.

Reply:
I like the idea! However, our PO wanted us to implement storing the "team size" right during event-creation. Right now, we don't really know when the team-assignment will happen during the process. Therefore, I would leave it as-is for now, and concentrate on getting a working solution until we have further details.